templates/new-design/parts/header.php

```diff
@@ -1,4 +1,52 @@
 <?php
+namespace Wporg\TranslationEvents\Templates\NewDesign\Parts;
 
-echo 'header<br>';
-// TODO.
+use Wporg\TranslationEvents\Templates;
+use Wporg\TranslationEvents\Urls;
+
+/** @var string $html_title */
+/** @var string|callable $page_title */
+/** @var string $url */
+/** @var string $image_url */
+/** @var string $html_description */
+/** @var ?callable $sub_head */
+/** @var ?string[] $breadcrumbs */
+
+$html_title       = $html_title ?? esc_html__( 'Translation Events', 'gp-translation-events' );
+$url              = $url ?? Urls::events_home();
+$html_description = $html_description ?? esc_html__( 'WordPress Translation Events', 'gp-translation-events' );
+$image_url        = $image_url ?? Urls::event_default_image();
+
+gp_title( $html_title );
+Templates::part( 'site-header', get_defined_vars() );
+Templates::part( 'breadcrumbs', array( 'extra_items' => $breadcrumbs ?? array() ) );
+
+?>
+
+<div class="event-list-top-bar">
```

Comment:
I think it could make sense to remove this from this PR for now, as it doesn't currently adhere to the new design, and implement it in a later PR with the new design.